common/LongPathHandler.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+//ToExtendedPath converts short paths to an extended path.
```

Comment:
Probably worth a comment here to explain that, as per  [https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file#maximum-path-length-limitation](https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file#maximum-path-length-limitation), "File I/O functions in the Windows API convert "/" to "\" as part of converting the name to an NT-style name, except when using the "\\?\" prefix as detailed in the following sections." THerefore, when converting to an extended path we MUST convert to use standard Windows path separators